doc/usecases/polyculture/suggest_groups.md

```diff
@@ -0,0 +1,31 @@
+# Use Case: Group suggestions
+
+## Summary
+
+- **Scope:** Plants Layer
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user is provided with suggestions on groups of plants to use.
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  - The user has opened the app.
+  - At least some of these layers are filled with data:
+    - Hydrology
+    - Terrain
+    - Shadows
+    - Soil
+    - Trees
```

Comment:
```suggestion
```

Reply:
I don't understand this change suggestion since the other one is to remove the whole precondition.

Reply:
You can ignore it. It was just for the case the precondition is needed for something else.